pulp_smash/tests/rpm/api_v2/test_orphan_remove.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+"""Test the removal of rpm orphans.
+
+This module assumes that the tests in
+:mod:`pulp_smash.tests.platform.api_v2.test_repository` and
++`pulp_smash.tests.platform.api_v2.test_sync_publish` hold true.
```

Comment:
This line is syntactically and semantically broken. But the automatic linters don't warn you, because this module isn't currently included in the documentation. This is what it takes to include it:

```diff
diff --git a/docs/api.rst b/docs/api.rst
index 1be39bd..79a7886 100644
--- a/docs/api.rst
+++ b/docs/api.rst
@@ -44,6 +44,7 @@ developers, not a gospel.
     api/pulp_smash.tests.rpm.api_v2.test_broker
     api/pulp_smash.tests.rpm.api_v2.test_duplicate_uploads
     api/pulp_smash.tests.rpm.api_v2.test_iso_crud
+    api/pulp_smash.tests.rpm.api_v2.test_orphan_remove
     api/pulp_smash.tests.rpm.api_v2.test_remove_unit
     api/pulp_smash.tests.rpm.api_v2.test_retain_old_count
     api/pulp_smash.tests.rpm.api_v2.test_sync_publish
diff --git a/docs/api/pulp_smash.tests.rpm.api_v2.test_orphan_remove.rst b/docs/api/pulp_smash.tests.rpm.api_v2.test_orphan_remove.rst
new file mode 100644
index 0000000..ad35039
--- /dev/null
+++ b/docs/api/pulp_smash.tests.rpm.api_v2.test_orphan_remove.rst
@@ -0,0 +1,7 @@
+`pulp_smash.tests.rpm.api_v2.test_orphan_remove`
+================================================
+
+Location: :doc:`/index` → :doc:`/api` →
+:doc:`/api/pulp_smash.tests.rpm.api_v2.test_orphan_remove`
+
+.. automodule:: pulp_smash.tests.rpm.api_v2.test_orphan_remove
```

Yes, this is a pain in the butt, and I'd like to have documentation stubs generated automatically. See #18. (I'll do this in a follow-up commit if you don't have time.)